extensions/resource-deployment/package.json

```diff
@@ -491,23 +514,79 @@
             "when": true
           }
         ],
-        "agreement": {
-          "template": "%azure-sqlvm-agreement%",
-          "links": [
-            {
-              "text": "%microsoft-privacy-statement%",
-              "url": "https://go.microsoft.com/fwlink/?LinkId=853010"
-            },
-            {
-              "text": "%azure-sqlvm-agreement-sqlvm-eula%",
-              "url": "https://azure.microsoft.com/support/legal/"
-            },
-            {
-              "text": "%azure-sqlvm-agreement-azdata-eula%",
-              "url": "https://aka.ms/eula-azdata-en"
-            }
-          ]
-        }
+        "agreement": [
+          {
+            "template": "%azure-sqlvm-agreement%",
+            "links": [
+              {
+                "text": "%microsoft-privacy-statement%",
+                "url": "https://go.microsoft.com/fwlink/?LinkId=853010"
+              },
+              {
+                "text": "%azure-sqlvm-agreement-sqlvm-eula%",
+                "url": "https://azure.microsoft.com/support/legal/"
+              },
+              {
+                "text": "%azure-sqlvm-agreement-azdata-eula%",
+                "url": "https://aka.ms/eula-azdata-en"
+              }
+            ],
+            "when": "true"
+          }
+        ]
+      },
+      {
+        "name": "azure-sql-mi",
+        "displayIndex": 6,
+        "displayName": "%azure-sql-mi-display-name%",
+        "description": "%azure-sql-mi-display-description%",
+        "platforms": "*",
+        "icon": "./images/miaa.svg",
```

Comment:
This shouldn't be MIAA - that's Azure Arc specifically. If we want to re-use the same icon I guess that's ok but you should still have the image name be managed-instance.svg or something